singer_sdk/typing.py

```diff
@@ -10,6 +10,15 @@
 
         Property("id", IntegerType, required=True),
         Property("foo_or_bar", StringType, allowed_values=["foo", "bar"]),
+        Property(
+            "permissions",
+            ArrayType(
+                th.StringType(
```

Comment:
It was my bad to bring `th.` prefix here 🙈

Reply:
Good catch, fixed: 905a77eb